main.cpp

```diff
@@ -161,8 +180,20 @@ void Bank::processTransactions()
     std::cout << "Money Withdrawn from ACCT: " << moneyWithdrawn << std::endl;
 }
 
-float Bank::produceCashBalance(float balance)
+float Bank::produceCashBalance(float balance = 0)
 {
+    for (balance = moneyDeposited; balance > 0; balance -= moneyWithdrawn)
+    {
+        std::cout << "Balance is: " << balance << std::endl;
+        std::cout << "$543.69 WithDrawn " << std::endl;
+        if (moneyWithdrawn > balance)
+        {
+        std::cout << "Not Enough Funds!!!!!" <<std::endl;
+        }
+    }
```

Comment:
- Fix this indentation
- default function arguments should be added where the function is declared, not where it is defined
- [ ] revise